### DISCUSSION TITLE
rbatis怎么映射一对一这种关联关系

### DISCUSSION THREAD

**Discussion by Clownsw** [rbatis/rbatis#257] [General]
![捕获](https://user-images.githubusercontent.com/28394742/188266927-6c858580-5ac0-4b9d-90ad-6a29099ba689.PNG)
如图 rbatis 怎么处理 `BlogTag` 内的 `Tag` 字段

**Comment by zhuxiujia** [comment]
> ![捕获](https://user-images.githubusercontent.com/28394742/188266927-6c858580-5ac0-4b9d-90ad-6a29099ba689.PNG) 如图 rbatis 怎么处理 `BlogTag` 内的 `Tag` 字段

参考项目[abs_admin](https://github.com/rbatis/abs_admin) 中关联表的操作，例如UserRole表
